src/main/java/com/sparta/spangeats/domain/menu/entity/Menu.java

```diff
@@ -0,0 +1,55 @@
+package com.sparta.spangeats.domain.menu.entity;
+
+import com.sparta.spangeats.common.Timestamped;
+import com.sparta.spangeats.domain.store.entity.Store;
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.Setter;
+
+@Getter
+@Setter
+@NoArgsConstructor
+@Entity
+@Table(name = "menus")
+public class Menu extends Timestamped {
+
+        @Id
+        @GeneratedValue(strategy = GenerationType.IDENTITY)
+        private Long id;
+
+        @Column(nullable = false)
+        private String name;
+
+        @Column(nullable = false)
+        private Integer price;
+
+        @Enumerated(EnumType.STRING)
+        private MenuStatus status;
+
+        @ManyToOne
+        @JoinColumn(name = "store_id", insertable = false, updatable = false)
+        private Store store;  // 가게와의 연관관계
+
+        @Column(nullable = false)
+        private String description;  // 메뉴 설명
+
+        // 생성자
+        public Menu(Store store, String name, Integer price, MenuStatus status, String description) {
+                this.store = store;
+                this.name = name;
+                this.price = price;
+                this.status = status;
+                this.description = description;
+        }
+
+        // 수정용 생성자 (id 포함)
+        public Menu(Long id, Store store, String name, Integer price, MenuStatus status, String description) {
+                this.id = id;
+                this.store = store;
+                this.name = name;
+                this.price = price;
+                this.status = status;
+                this.description = description;
+        }
```

Comment:
수정용이라면 아예 메서드 이름으로 명확하게 해주는 것도 좋을 거 같습니다.!! 그리고 id가 변경될 일은 없겠죵?!?! 그래서 빼두었습니다.
```suggestion
        public void update(Store store, String name, Integer price, MenuStatus status, String description) {
                this.store = store;
                this.name = name;
                this.price = price;
                this.status = status;
                this.description = description;
        }
```